packages/ifpack2/src/Ifpack2_RILUK_def.hpp

```diff
@@ -307,6 +310,11 @@ setParameters (const Teuchos::ParameterList& params)
     getParamTryingTypes<magnitude_type, magnitude_type, double>
       (relaxValue, params, paramName, prefix);
   }
+  {
+    const std::string paramName ("fact: iluk overalloc");
```

Comment:
I was unsure how to test these changes without a parameter.  For runtime efficiency and backward compatibility with respect to runtime performance, I wanted a heuristic that would rarely invoke the resizing.  But I needed a way to invoke the resizing in testing.  The parameter allowed me to do both.  I set the default value to something realistic that rarely needs resizing.  But for testing I set it artificially small so that resizing gets invoked and the loop gets exercised.  I am open to other suggestions, but there did not seem to be a consensus in the meeting last week on how to design a test problem that would fail with the original code.

Reply:
@kddevin that makes total sense to me -- thanks for explaining! :-D